static/home/css/style.css

```diff
@@ -389,6 +389,9 @@ nav{
 	height: 2px;
 	background-color: #ff8c00;
 }
+#initiatives .container{
+	padding-bottom: 15px;
```

Comment:
Instead of adding `padding-bottom` to `container`, I think it would be better if we add `padding-bottom: 25px` to `#initiatives .glide` as glide bullets have absolute positioning w.r.t `.glide` and hence, adding a `padding-bottom` to it will save us from changing default `bottom` property of `.glide_bullets`.